### PR TITLE
Updating to latest released agent (ciprod10162018-2)

### PR DESCRIPTION
**Pull request by vishiy**
#### Which issue this PR fixes
*(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close that issue when PR gets merged)*
  - fixes #

#### Checklist
[Place an '[x]' (no spaces) in all applicable fields. Please remove unrelated fields.]
- [x] [DCO](https://www.helm.sh/blog/helm-dco/index.html) signed
- [x] Chart Version bumped
- [x] Variables are documented in the README.md

### CODE REVIEW

**Review by davidkarlsen — APPROVED** [review]

File: incubator/azuremonitor-containers/Chart.yaml [hunk 1]
```diff
@@ -2,7 +2,7 @@ apiVersion: v1
 appVersion: 2.0.0-3
```
Comment (davidkarlsen) [inline incubator/azuremonitor-containers/Chart.yaml:2]: should this be updated as well?

Comment (vishiy) [reply incubator/azuremonitor-containers/Chart.yaml:2]: No. We have versions for all our components.

**Comment by vishiy** [comment]
@davidkarlsen - who can merge ?

**Pull request merged**